api/v1/services/user.py

```diff
@@ -362,6 +362,11 @@ def follow_user(self, db: Session, user_id: str, user: User):
         if followee not in user.followings:
             user.followings.append(followee)
 
+            notification2 = Notification(
```

Comment:
Why is there `notification2` here and what is been added to the database is `notification`. Sele dey whine me😂😂